client/extensions/woocommerce/app/orders/orders-list.js

```diff
@@ -111,42 +121,22 @@ class Orders extends Component {
 		);
 	}
 
-	onPageClick = ( i ) => {
-		return () => {
-			this.props.setCurrentPage( this.props.siteId, i );
-		};
-	}
-
-	renderPageLink = ( i ) => {
-		// We want this to start at 1, not 0
-		i++;
-		return (
-			<li key={ i }>
-				{ ( i !== this.props.currentPage )
-					? <Button compact borderless onClick={ this.onPageClick( i ) }>{ i }</Button>
-					: <span>{ i }</span>
-				}
-			</li>
-		);
-	}
-
-	renderPagination = () => {
-		const { totalPages } = this.props;
-		// @todo Bring back pagination
-		if ( true || totalPages < 2 ) {
-			return null;
-		}
-
-		return (
-			<ul>
-				{ times( totalPages, this.renderPageLink ) }
-			</ul>
-		);
+	onPageClick = page => {
+		this.props.setCurrentQuery( this.props.siteId, { page } );
```

Comment:
No, if you look at the reducer that handles `currentPage` and `currentStatus`, they're updated separately only if the new value exists. You can see this working in the tests, too. I can see how calling it `set-` sounds like it should set the whole thing, do you think `updateCurrentOrdersQuery` would be more clear?